js/json-parser.js

```diff
@@ -0,0 +1,58 @@
+"use strict";
+
+(function (exports) {
+  exports.JsonParser = function () {
+    var _url = [];
+    var _title = [];
+    var _body = [];
+    var _thumbnail = [];
+
+    var _iterateThroughJson = function (out) {
+      for (var i = 0; i < out.response.results.length; i++) {
+        _url[i] = out.response.results[i].webUrl;
+        _title[i] = out.response.results[i].webTitle;
+        _body[i] = out.response.results[i].fields.body;
+        _thumbnail[i] = out.response.results[i].fields.thumbnail;
+      }
+    }
+
+    var parseJsonFromUrl = function (url, callback) {
+      fetch(url).then(res => res.json()).then((out) => {
```

Comment:
I'm not sure why you have both a parseJsonFromUrl and a parseJsonFromVar, but I would expect that parsing and fetching should be two different responsibilities handled by two different classes.